### PR TITLE
added the required attributes - rhidp-3968

### PR DESCRIPTION
**Pull request by pabel-rh**
PR title format: Adding attributes for Installation Guides

IMPORTANT: Do Not Merge - To be merged by Docs Team Only

**Version(s):** 1.2, 1.3

**Issue:** RHIDP-3968

**Reviews:**
- [x] Docs review: @hmanwani-rh

### CODE REVIEW

File: artifacts/attributes.adoc [hunk 1]
```diff
@@ -67,3 +67,18 @@
 
 :LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/configuring_plugins_in_red_hat_developer_hub/index
 :NameOfPluginsGuide: Configuring plugins in {product}
+
+:authentication-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:71]:
```suggestion
:install-on-ocp-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
```

File: artifacts/attributes.adoc [hunk 2]
```diff
@@ -67,3 +67,18 @@
...
+:authentication-book-title: Installing Red Hat Developer Hub on OpenShift Container Platform
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:72]:
```suggestion
:install-on-ocp-title: Installing Red Hat Developer Hub on OpenShift Container Platform
```

File: artifacts/attributes.adoc [hunk 3]
```diff
@@ -67,3 +67,18 @@
...
+
+:authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:74]:
```suggestion
:install-on-eks-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
```

File: artifacts/attributes.adoc [hunk 4]
```diff
@@ -67,3 +67,18 @@
...
+:authorization-book-title: Installing Red Hat Developer Hub on Amazon Elastic Kubernetes Service
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:75]:
```suggestion
:install-on-eks-title: Installing Red Hat Developer Hub on Amazon Elastic Kubernetes Service
```

File: artifacts/attributes.adoc [hunk 5]
```diff
@@ -67,3 +67,18 @@
...
+
+:authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:77]:
```suggestion
:install-on-aks-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
```

File: artifacts/attributes.adoc [hunk 6]
```diff
@@ -67,3 +67,18 @@
...
+:authorization-book-title: Installing Red Hat Developer Hub on Microsoft Azure Kubernetes Service
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:78]:
```suggestion
:install-on-aks-title: Installing Red Hat Developer Hub on Microsoft Azure Kubernetes Service
```

File: artifacts/attributes.adoc [hunk 7]
```diff
@@ -67,3 +67,18 @@
...
+
+:authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:80]:
```suggestion
:install-on-airgap-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
```

File: artifacts/attributes.adoc [hunk 8]
```diff
@@ -67,3 +67,18 @@
...
+:authorization-book-title: Installing Red Hat Developer Hub in an air-gapped environment
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:81]:
```suggestion
:install-on-airgap-title: Installing Red Hat Developer Hub in an air-gapped environment
```

File: artifacts/attributes.adoc [hunk 9]
```diff
@@ -67,3 +67,18 @@
...
+
+:authorization-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:83]:
```suggestion
:upgrade-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
```

File: artifacts/attributes.adoc [hunk 10]
```diff
@@ -67,3 +67,18 @@
...
+:authorization-book-title: Upgrading Red Hat Developer Hub
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:84]:
```suggestion
:upgrade-book-title: Upgrading Red Hat Developer Hub
```

**Review by hmanwani-rh — CHANGES_REQUESTED** [review]
@pabel-rh The attribute names should be unique and relevant to their values. I've provided some suggestions below. Please feel free to reach out if you have any questions.

File: artifacts/attributes.adoc [hunk 11]
```diff
@@ -67,3 +67,18 @@
 
 :LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/configuring_plugins_in_red_hat_developer_hub/index
 :NameOfPluginsGuide: Configuring plugins in {product}
+
+:install-on-ocp-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
+:install-on-ocp-book-title: Installing Red Hat Developer Hub on OpenShift Container Platform
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:72]:
```suggestion
:install-on-ocp-book-title: Installing {product} on {ocp-short}
```

File: artifacts/attributes.adoc [hunk 12]
```diff
@@ -67,3 +67,18 @@
...
+
+:install-on-eks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
+:install-on-eks-book-title: Installing Red Hat Developer Hub on Amazon Elastic Kubernetes Service
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:75]:
```suggestion
:install-on-eks-book-title: Installing {product} on {eks-brand-name}
```

File: artifacts/attributes.adoc [hunk 13]
```diff
@@ -67,3 +67,18 @@
...
+
+:install-on-aks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
+:install-on-aks-book-title: Installing Red Hat Developer Hub on Microsoft Azure Kubernetes Service
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:78]:
```suggestion
:install-on-aks-book-title: Installing {product} on {aks-brand-name}
```

File: artifacts/attributes.adoc [hunk 14]
```diff
@@ -67,3 +67,18 @@
...
+
+:install-in-air-gap-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
+:install-in-air-gap-book-title: Installing Red Hat Developer Hub in an air-gapped environment
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:81]:
```suggestion
:install-in-air-gap-book-title: Installing {product} in an air-gapped environment
```

File: artifacts/attributes.adoc [hunk 15]
```diff
@@ -67,3 +67,18 @@
...
+
+:upgrade-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
+:upgrade-book-title: Upgrading Red Hat Developer Hub
```
Comment (hmanwani-rh) [inline artifacts/attributes.adoc:84]:
```suggestion
:upgrade-book-title: Upgrading {product}
```

**Review by hmanwani-rh — CHANGES_REQUESTED** [review]
@pabel-rh Apologies for the oversight in my previous review; I forgot to mention that we need to include attributes for product names and partner platforms.
The rest looks good.

**Review by hmanwani-rh — APPROVED** [review]

File: artifacts/attributes.adoc [hunk 16]
```diff
@@ -67,3 +67,18 @@
...
+:install-on-eks-book-title: Installing {product} on {eks-brand-name}
+
+:install-on-aks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
+:install-on-aks-book-title: Installing {product} on {aks-brand-name}
+
+:install-in-air-gap-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
+:install-in-air-gap-book-title: Installing {product} in an air-gapped environment
+
+:upgrade-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
+:upgrade-book-title: Upgrading {product}
```
Comment (themr0c) [inline artifacts/attributes.adoc:84]: Consider keeping the gerund and adding the Authentication and Authorization books:

```suggestion

:installing-on-ocp-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_openshift_container_platform/index
:installing-on-ocp-book-title: Installing {product} on {ocp-short}

:installing-on-eks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_amazon_elastic_kubernetes_service/index
:installing-on-eks-book-title: Installing {product} on {eks-brand-name}

:installing-on-aks-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_on_microsoft_azure_kubernetes_service/index
:installing-on-aks-book-title: Installing {product} on {aks-brand-name}

:installing-in-air-gap-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index
:installing-in-air-gap-book-title: Installing {product} in an air-gapped environment

:authentication-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authentication/index
:authentication-book-title: Authentication

:authorization-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authorization/index
:authorization-book-title: Authorization

:upgrading-book-url: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/upgrading_red_hat_developer_hub/index
:upgrading-book-title: Upgrading {product}
```

**Review by themr0c — APPROVED** [review]

**Comment by themr0c** [comment]
@pabel-rh The automatic cherry-pick to the 1.2.x branch failed. Can you cherry-pick manually?

File: modules/dynamic-plugins/con-ansible-plugin.adoc
Comment (Gerry-Forde) [inline modules/dynamic-plugins/con-ansible-plugin.adoc:1]: @themr0c @pabel-rh This fix also needs to be cherrypicked to 1.2.x

Comment (pabel-rh) [reply modules/dynamic-plugins/con-ansible-plugin.adoc:1]: @themr0c , could you help cherrypick this PR to 1.2.x as well?

File: artifacts/attributes.adoc [hunk 17]
```diff
@@ -67,3 +67,24 @@
 
 :LinkPluginsGuide: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/configuring_plugins_in_red_hat_developer_hub/index
 :NameOfPluginsGuide: Configuring plugins in {product}
+
+:authentication-book-url: https://docs.redhat.com/documentation/en-us/red_hat_developer_hub/{product-version}/html-single/authentication/index
```
Comment (gwynnemonahan) [inline artifacts/attributes.adoc:71]: Where does `/us-en/` come from?

https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/authentication/index works as does https://docs.redhat.com/us-en/documentation/red_hat_developer_hub/1.2/html/authentication/index and https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/authentication/index


https://docs.redhat.com/us-en/documentation/red_hat_developer_hub/1.2/html-single/authentication/index returns a 404.

I wonder if removing `/us-en/` would resolve the issue. `/us-en/` is not present in any of the other URLs in this PR.

Comment (hmanwani-rh) [reply artifacts/attributes.adoc:71]: @themr0c ^

**Pull request merged**